components/form/builder/src/index.js

```diff
@@ -53,52 +53,56 @@ const FormBuilder = ({
     return nextFields
   }
 
-  const handlerChange = useCallback(async (id, value, extraParams) => {
-    const reducerWithRules = reducer(
-      rules,
-      formID,
-      baseAPIURL,
-      responseInterceptor,
-      requestInterceptor,
-      locale,
-      extraParams
-    )
-    const timerShowSpinner = setTimeout(
-      () => setStateShowSpinner(true),
-      FormBuilder.USER_MINIMAL_DELAY
-    )
-    const transformedValue = transformations(
-      id,
-      value,
-      pickFieldById(fields, id)
-    )
+  const handlerChange = useCallback(
+    async (id, value, extraParams, autocompletedField = false) => {
```

Comment:
There's a lot of lines change because the new tabulation, so I'm pointing the real changes.

Here, `onChange` receives a new param `autocompletedField`, `false` by default.